.github/workflows/development-environment.yml

```diff
@@ -37,43 +36,74 @@ jobs:
 
       # Trick for unattended Docker installations
       # https://github.com/docker/for-mac/issues/2359#issuecomment-943131345
-      - name: Install prerequisites
+      # NOTE: This can sometimes take up to 10 minutes
+      - name: Install Docker
         run: |
-          HOMEBREW_NO_AUTO_UPDATE=1 brew install --cask docker
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install -v --cask docker
           sudo /Applications/Docker.app/Contents/MacOS/Docker --unattended --install-privileged-components
           open -a /Applications/Docker.app --args --unattended --accept-license
-          make prerequisites
```

Comment:
This was used to install libxmlsec1 and pyenv. We don't need *both* packages in the same job.